packages/tldraw/src/lib/shapes/embed/EmbedShapeUtil.tsx

```diff
@@ -38,7 +39,8 @@ export class EmbedShapeUtil extends BaseBoxShapeUtil<TLEmbedShape> {
 		return true
 	}
 	override canResize(shape: TLEmbedShape) {
-		return !!getEmbedInfo(shape.props.url)?.definition?.doesResize
+		const definitions = useEmbedDefinitions()
+		return !!getEmbedInfo(definitions, shape.props.url)?.definition?.doesResize
```

Comment:
If these are used together, it might be better to make a `useEmbedDefinition` hook?